<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Доступные теги:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Известные проигрыватели:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Использовать MPRIS v.2</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="_2">
        <item>
         <widget class="QPushButton" name="btnPrev">
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPlay">
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnStop">
          <property name="text">
           <string>[]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNext">
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="Memo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>%artist - %title &lt; %album &gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btnGetMetaData">
      <property name="text">
       <string>Получить данные</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>playerChange(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setPlayerVersion(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPlay</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>playerPlay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>playerStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPrev</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>playerPrev()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNext</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>playerNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnGetMetaData</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>getMetadata()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getMetadata()</slot>
  <slot>playerChange(QString)</slot>
  <slot>setPlayerVersion(bool)</slot>
  <slot>playerPlay()</slot>
  <slot>playerStop()</slot>
  <slot>playerPrev()</slot>
  <slot>playerNext()</slot>
 </slots>
</ui>
